discord_bot/discord_bot.py

```diff
@@ -144,24 +142,45 @@ def load_config(config_file_path):
                     self.logger.warning("Invalid Session")
                 elif message["op"] == Opcodes.DISPATCH:
                     event = message["t"]
-                    if event == Events.READY:
+                    if event == Events.READY.value:
                         self.session_id = message["d"]["session_id"]
+                    elif event == Events.MESSAGE_CREATE.value:
+                        channel_id = message["d"]["channel_id"]
```

Comment:
I think I'd rather the separate function take the raw message completely and parse it itself to save as many lines as possible in this mega-function.